build fails - new cache does not exist

Seems[/tmp/.buildx-cache-new](https://github.com/os-observability/tempo-operator/pull/18/files) does not exist.

CI (main): https://github.com/os-observability/tempo-operator/actions/runs/3050914986/jobs/4918544129#step:12:9

```
Run rm -rf /tmp/.buildx-cache
mv: cannot stat '/tmp/.buildx-cache-new': No such file or directory
Error: Process completed with exit code 1.
```

cc @iblancasa

--- a/.github/workflows/publish-images.yaml
+++ b/.github/workflows/publish-images.yaml
@@ -67,7 +67,7 @@ jobs:
           tags: ${{ steps.docker_meta.outputs.tags }}
           labels: ${{ steps.docker_meta.outputs.labels }}
           cache-from: type=local,src=/tmp/.buildx-cache
-          cache-to: type=local,dest=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new,mode=max
       -
         # Temp fix
         # https://github.com/docker/build-push-action/issues/252

